build/webpack.npm.config.js

```diff
@@ -30,7 +30,10 @@ module.exports = {
             {
                 test: /\.js$/,
```

Comment:
This file is only used to generate CSS, there is not webpack for JS for npm packages

Reply:
In other words webpack is not used for anything other than making wrapper libraries.